cinema_service/settings.py

```diff
@@ -86,8 +88,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.getenv("POSTGRES_DB", "default_db_name"),
```

Comment:
Good job on using environment variables for your database settings. This allows you to have different settings for different environments and keeps sensitive information out of your code.